tests/test_abundance.py

```diff
@@ -0,0 +1,135 @@
+# %%# -*- coding: utf-8 -*-
+import pandas as pd
+import numpy as np
+import pytest
+from proteometer.abundance import prot_abund_correction_matched
+
+
+# %%
+def test_protein_abund_correction_matched():
```

Comment:
I think it's better to have separate test functions then to have one that defines others in an internal scope... unless there's some reason to do it this way that I don't know about.